Classes/EventListener/EventListenerLocator.php

```diff
@@ -63,14 +63,13 @@ public function initializeObject()
     }
 
     /**
-     * Returns event listeners for the given event (type)
+     * Returns event listeners for the given event type
      *
-     * @param EventInterface $event
+     * @param string $eventType
      * @return \callable[]
      */
-    public function getListeners(EventInterface $event): array
+    public function getListeners(string $eventType): array
```

Comment:
not exactly related but these methods were only used in one place each where we already know the event type